ts/components/conversation/message/reactions/ReactionPopup.tsx

```diff
@@ -75,53 +77,71 @@ const generateContacts = async (messageId: string, senders: Array<string>) => {
       results.splice(meIndex, 1);
       results = [window.i18n('you'), ...results];
     }
+    if (results && results.length > 0) {
+      return results;
+    }
   }
-  return results;
+  return null;
 };
 
-const Contacts = (contacts: string) => {
-  if (!contacts) {
+const Contacts = (contacts: Array<string>, count: number) => {
+  if (!Boolean(contacts?.length > 0)) {
     return;
   }
 
-  if (contacts.includes('&') && contacts.includes('other')) {
-    const [names, others] = contacts.split('&');
+  const reactors = contacts.length;
+  if (reactors === 1 || reactors === 2 || reactors === 3) {
+    return (
+      <span>
+        {window.i18n(
+          reactors === 1
+            ? 'reactionPopupOne'
+            : reactors === 2
+            ? 'reactionPopupTwo'
+            : 'reactionPopupThree',
+          contacts
+        )}{' '}
+        {window.i18n('reactionPopup')}
```

Comment:
this actually won't work with the tools to detect unused strings but we will see later